doc/guides/maintaining-c-ares.md

```diff
@@ -0,0 +1,63 @@
+# Maintaining c-ares
+
+Updates to the c-ares dependency involve the following steps:
+1. Downloading the source archive for the new version.
+1. Unpacking the source in a temporary workspace directory.
+1. Removing the `test` directory (to save disk space).
+1. Copying over the existing `.gitignore`, pre-generated `config` directory and
```

Comment:
```suggestion
2. Unpacking the source in a temporary workspace directory.
3. Removing the `test` directory (to save disk space).
4. Copying over the existing `.gitignore`, pre-generated `config` directory and
```

Reply:
I don't understand why we want to do that. It's a feature of markdown to write ordered lists only with ones, and I like it because it allows to add an item anywhere in the list without having te renumber everything after it.

Reply:
I've run the formatter and committed the results (plus the comma change and reworded the bit about LICENSE updates). I kind of agree with @targos re. the GitHub flavored markdown feature but I'm not going to fight the formatter.

Reply:
@targos @richardlau This is a bit of a spaces vs. tabs kind of thing I suppose, but for what it's worth, I think it's better to format it this way so that the raw markdown is more easily readable. It's supposed to be one of the advantages of markdown that it is useable/understandable in its raw format. It makes things slightly harder for the author, but I think when the differences are small, we should favor the reader. (That said, I haven't looked to see if there is a configuration option in remark to change this behavior. There might be.)

Reply:
(And I also don't actually feel strongly about it. We do more writing markdown than reading the raw markdown probably, so... I'm just going to submit to the formatter... ¯\\_(ツ)_/¯ )